src/tableset.h

```diff
@@ -39,8 +39,8 @@ template<class T>
 class NUT_EXPORT TableSet : public TableSetBase
 {
 public:
-    explicit TableSet(Database *parent = nullptr);
```

Comment:
Why optional parameter of constructor removed?

Reply:
MSVC not like this I don't know Why !!!